crates/runtime/src/lib.rs

```diff
@@ -296,6 +299,7 @@ pub struct Runtime {
     datasets_health_monitor: Option<Arc<DatasetsHealthMonitor>>,
     metrics_endpoint: Option<SocketAddr>,
     prometheus_registry: Option<prometheus::Registry>,
+    tokio_servers_runtime: Option<Arc<TokioRuntime>>,
```

Comment:
Is prefixing with `tokio` the best move? If we're already in a `tokio` runtime then `servers_runtime` or `servers_tokio_runtime` might be a bit clearer.

Reply:
This crate is called `runtime` with a `Runtime` struct, so I wanted to make it clear this was specific to tokio